endpoints/v2/v2auth.py

```diff
@@ -139,7 +139,7 @@ def generate_registry_jwt(auth_result):
     token = generate_bearer_token(
         audience_param, subject, context, access, TOKEN_VALIDITY_LIFETIME_S, instance_keys
     )
-    return jsonify({"token": token.decode("ascii")})
+    return jsonify({"token": token})
```

Comment:
I'm getting an exception on this line when I try to login after just having rebuilt quay locally - did I miss a step to successfully run this?

```
2022-08-08 12:36:26,547 [229] [ERROR] [gunicorn.error] Error handling request /v2/auth?account=admin&service=quay%3A8080
Traceback (most recent call last):
  File "/app/lib/python3.9/site-packages/gunicorn/workers/base_async.py", line 55, in handle
    self.handle_request(listener_name, req, client, addr)
  File "/app/lib/python3.9/site-packages/gunicorn/workers/ggevent.py", line 127, in handle_request
    super().handle_request(listener_name, req, sock, addr)
  File "/app/lib/python3.9/site-packages/gunicorn/workers/base_async.py", line 108, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/app/lib/python3.9/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/app/lib/python3.9/site-packages/werkzeug/middleware/proxy_fix.py", line 169, in __call__
    return self.app(environ, start_response)
  File "/app/lib/python3.9/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/app/lib/python3.9/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/app/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/app/lib/python3.9/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/app/lib/python3.9/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/app/lib/python3.9/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/app/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/app/lib/python3.9/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/lib/python3.9/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/quay-registry/auth/decorators.py", line 65, in wrapper
    return func(*args, **kwargs)
  File "/quay-registry/util/cache.py", line 36, in add_no_cache
    response = f(*args, **kwargs)
  File "/quay-registry/endpoints/decorators.py", line 154, in wrapper
    return func(*args, **kwargs)
  File "/quay-registry/endpoints/v2/v2auth.py", line 142, in generate_registry_jwt
    return jsonify({"token": token})
  File "/app/lib/python3.9/site-packages/flask/json/__init__.py", line 370, in jsonify
    dumps(data, indent=indent, separators=separators) + "\n",
  File "/app/lib/python3.9/site-packages/flask/json/__init__.py", line 211, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/usr/lib64/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib64/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib64/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/app/lib/python3.9/site-packages/flask/json/__init__.py", line 100, in default
    return _json.JSONEncoder.default(self, o)
  File "/usr/lib64/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
```

Reply:
Hmm, shouldn't need to do anything. Let me double check, maybe I have something cached.

Reply:
yeah something was wrong on my end, after a clean cache then build it now works 🎉